docs/devel/update-release-docs.md

```diff
@@ -0,0 +1,145 @@
+  (https://github.com/kubernetes/kubernetes/blob/gh-pages/_layouts/docwithnav.html)
+  to include this new navigation html file. Example PR: [#16143](https://github.com/kubernetes/kubernetes/pull/16143).
+* [Pull docs from release branch](#updating-docs-in-gh-pages-branch) in _vX.Y
+  folder.
+
+Once these changes have been submitted, you should be able to reach the docs at
+http://kubernetes.io/vX.Y/docs/ where you can test them.
+
+To make X.Y the default version of docs:
+
+* Update [_config.yml](https://github.com/kubernetes/kubernetes/blob/gh-pages/_config.yml)
+  and [_docs/index.md](https://github.com/kubernetes/kubernetes/blob/gh-pages/_docs/index.md)
+  to point to the new version. Example PR: [#16416](https://github.com/kubernetes/kubernetes/pull/16416).
+* Update [_includes/docversionselector.html](https://github.com/kubernetes/kubernetes/blob/gh-pages/_includes/docversionselector.html)
+  to make vX.Y the default version.
+* Add "Disallow: /vX.Y-1/" to existing [robots.txt](https://github.com/kubernetes/kubernetes/blob/gh-pages/robots.txt)
+  file to hide old content from web crawlers and focus SEO on new docs. Example PR:
+  [#16388](https://github.com/kubernetes/kubernetes/pull/16388).
+* Regenerate [sitemaps.xml](https://github.com/kubernetes/kubernetes/blob/gh-pages/sitemap.xml)
+  so that it now contains vX.Y links. Example PR: [#17126](https://github.com/kubernetes/kubernetes/pull/17126).
+* Update [_layouts/docwithnav.html] (https://github.com/kubernetes/kubernetes/blob/gh-pages/_layouts/docwithnav.html)
+  to include [_includes/archivedocnotice.html](https://github.com/kubernetes/kubernetes/blob/gh-pages/_includes/archivedocnotice.html)
+  for vX.Y-1 docs which need to be archived.
+
+http://kubernetes.io/docs/ should now be redirecting to http://kubernetes.io/vX.Y/docs/.
+
+# Updating docs in an existing collection
+
+The high level steps to update docs in an existing collection are:
+
+1. Update docs on HEAD (master branch)
+2. Cherryick the change in relevant release branch.
+3. Update docs on gh-pages.
+
+## Updating docs on HEAD
+
+[Development guide](development.md) provides general instructions on how to contribute to kubernetes github repo.
+[Docs how to guide](how-to-doc.md) provides conventions to follow while writting docs.
+
+## Updating docs in release branch
+
+Once docs have been updated in the master branch, the changes need to be
+cherrypicked in the latest release branch.
+[Cherrypick guide](cherry-picks.md) has more details on how to cherrypick your change.
+
+## Updating docs in gh-pages branch
+
+Once release branch has all the relevant changes, we can pull in the latest docs
+in gh-pages branch.
+Run the following 2 commands in gh-pages branch to update docs for release X.Y:
+
+```
+rm -rf _vX.Y/docs/ _vX.Y/examples/
+go run _tools/release_docs/main.go _tools/release_docs/api-reference-process.go --branch release-X.Y --output-dir _vX.Y --version vX.Y
```

Comment:
Just tried out this script again and it's working. I think it's better to use the script.